src/test/scala/firrtlTests/UnitTests.scala

```diff
@@ -47,11 +47,12 @@ class UnitTests extends FlatSpec with Matchers {
         |    input y: {a : UInt<1>}
         |    output x: {a : UInt<1>, b : UInt<1>}
         |    x <= y""".stripMargin
-    intercept[PassExceptions] {
+    val thrown = intercept[PassExceptions] {
       passes.foldLeft(Parser.parse("",input.split("\n").toIterator)) {
         (c: Circuit, p: Pass) => p.run(c)
       }
     }
+    assert(thrown.getMessage contains "Type mismatch. Cannot connect")
```

Comment:
I guess I wanted to make sure my updated message gets in, but I'll change this, now that I've manually checked